test/find_SUITE.erl

```diff
@@ -18,7 +18,7 @@
 %%%=============================================================================
 
 -spec all() -> [atom()].
-all() -> [find_by_sort, find_all_sort].
+all() -> [].  %% @todo recover the whole list of test cases when #257
```

Comment:
tiny detail: specify that the issue is inaka/sumo_db#257 and not #257 (in this repo)